AutomatticTracks/src/test/java/com/automattic/android/tracks/CrashLogging/CrashLoggingTest.kt

```diff
@@ -0,0 +1,70 @@
+package com.automattic.android.tracks.crashlogging
+
+import com.automattic.android.tracks.TracksUser
+import com.automattic.android.tracks.fakes.FakeDataProvider
+import com.automattic.android.tracks.fakes.testUser
+import org.junit.Test
+import org.junit.runner.RunWith
+import org.mockito.Mockito.never
+import org.mockito.Mockito.spy
+import org.mockito.Mockito.times
+import org.mockito.Mockito.verify
+import org.robolectric.RobolectricTestRunner
+import org.robolectric.RuntimeEnvironment
+
+@RunWith(RobolectricTestRunner::class)
+class CrashLoggingTest {
+
+    lateinit var dataProvider: CrashLoggingDataProvider
+    var user: TracksUser? = null
+
+    private fun initialize(currentUser: TracksUser? = null) {
+
+        user = if (currentUser != null) spy(currentUser) else null
+
+        dataProvider = spy(FakeDataProvider(currentUser = user))
+
+        CrashLogging.start(
+            context = RuntimeEnvironment.systemContext,
+            dataProvider = dataProvider
+        )
+    }
+
+    @Test(expected = UnsupportedOperationException::class)
+    fun shouldThrowExceptionOnRequest() {
+        CrashLogging.crash()
+    }
+
+    @Test
+    fun `should call for required arguments during initialisation`() {
```

Comment:
```suggestion
    fun `should call for required arguments during initialization`() {
```

Just so it's consistent with the `initialize` method name, even though both spelling seems to be correct. Feel free to ignore it if you end up changing the method name for `initialize`.